substrate/frame/fast-unstake/src/tests.rs

```diff
@@ -806,12 +806,8 @@ mod on_idle {
 			assert_ok!(FastUnstake::register_fast_unstake(RuntimeOrigin::signed(VALIDATOR_PREFIX)));
 
 			// but they indeed are exposed!
-			assert!(pallet_staking::EraInfo::<T>::get_paged_exposure(
-				BondingDuration::get() - 1,
-				&VALIDATOR_PREFIX,
-				0
-			)
-			.is_some());
+			let exposure = Staking::eras_stakers(BondingDuration::get() - 1, &VALIDATOR_PREFIX);
```

Comment:
maybe nicer here is to just do:

```
assert!(T::Staking::is_exposed_in_era(VALIDATOR_PREFIX, BondingDuration::get() - 1))
```

It seems like the impl of `is_exposed_in_era` defines being exposed as:

* either you are a nominator behind an exposure
* or you are the associated validator of that exposure (even if your `exposure.own == 0`)